src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -155,6 +155,11 @@ internal class CommonLocalizableStrings
         /// arguments
         public const string ArgumentsProjectOrSolutionDescription = "The project or solution to operation on. If a file is not specified, the current directory is searched.";
 
+        /// sln
+        public const string ArgumentsProjectDescription = "The project file to operate on. If a file is not specified, the command will search the current directory for one.";
+        public const string ArgumentsSolutionDescription = "Solution file to operate on. If not specified, the command will search the current directory for one.";
+        public const string CmdSlnFile = "<SLN_FILE>";
```

Comment:
Can you also add CmdProjectFile = "&lt;PROJECT&gt;"

In case we do need to use the localized string. I'd end up using it in dotnet add p2p